readme.md

```diff
@@ -53,7 +53,71 @@ Directory names should be lowercase. Filenames should be singular.
 *todo: evaluate common prefix like `pagelib` and BEM naming.*
 
 ## Development setup and workflow
-*Coming soon.*
+The Android and iOS Wikipedia apps make use of the `wikimedia-page-library`. At some point their respective build phases invoke `npm install` which causes a *published* version of this library to be retrieved and used in their respective builds. 
```

Comment:
Oh an interesting pattern taking hold in Swift lately is using `guard` statements for early exiting when some set of conditions is not met. One of the fallouts from this is reduced indentation - this article talks a bit about it: https://thatthinginswift.com/guard-statement-swift/